charts/splunk-connect-for-snmp/templates/worker/discovery/deployment.yaml

```diff
@@ -0,0 +1,67 @@
+{{- if eq (include "splunk-connect-for-snmp.discovery.enable" .) "true" }}
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: {{ include "splunk-connect-for-snmp.worker.fullname" . }}-discovery
+  labels:
+    {{- include "splunk-connect-for-snmp.worker.discovery.labels" . | nindent 4 }}
+spec:
+  replicas: 1
+  selector:
+    matchLabels:
+      {{- include "splunk-connect-for-snmp.worker.discovery.selectorLabels" . | nindent 6 }}
+  template:
+    metadata:
+      labels:
+        {{- include "splunk-connect-for-snmp.worker.discovery.selectorLabels" . | nindent 8 }}
+    spec:
```

Comment:
Don't we need to specify `hostNetwork: true` somewhere? Is this really work with nmapping external hosts?